fspiop-api/documents/API-Definition_v1.1.1.md

```diff
@@ -0,0 +1,5207 @@
+
+**Table 37 –- Version history for resource /bulkQuotes**
+
+#### 6.9.2 Service Details
+
+[Figure 59](#figure-59) shows how the bulk quotes process works, using the **POST /bulkQuotes** service. When receiving the bulk of transactions from the Payer, the Payer FSP should:
+
+1. Lookup the FSP in which each Payee is; for example, using the API Resource **/participants**, [Section 6.2](#62-api-resource-participants).
+
+2. Divide the bulk based on Payee FSP. The service **POST /bulkQuotes** is then used for each Payee FSP to get the bulk quotes from each Payee FSP. Each quote result will contain the ILP Packet and condition (see [Section 4.5](#45-ilp-packet) and [Section 4.4](#44-conditional-transfers)) needed to perform each transfer in the bulk transfer (see API Resource **/bulkTransfers**, [Section 6.10](#610-api-resource-bulktransfers)), which will perform the actual financial transaction from the Payer to each Payee.
+
+###### Figure 59
+
+{% uml src="assets/diagrams/sequence/figure59.plantuml" %}
+{% enduml %}
+
+**Figure 59 -- Example bulk quote process**
+
+#### 6.9.3 Requests
+
+This section describes the services that can be requested by a client in the API on the resource **/bulkQuotes**.
+
+#### 6.9.3.1 GET /bulkQuotes/_{ID}_
+
+Alternative URI: N/A
+
+Logical API service: **Retrieve Bulk Quote Information**
+
+The HTTP request **GET /bulkQuotes/**_{ID}_ is used to get information regarding a previously-created or requested bulk quote.
+
+The _{ID}_ in the URI should contain the **bulkQuoteId** (see [Table 38](#table-38)) that was used for the creation of the bulk quote.
+
+Callback and data model information for **GET /bulkQuotes/**_{ID}_:
+
+- Callback -- [**PUT /bulkQuotes/**_{ID}_](#6941-put-bulkquotesid)
+- Error Callback -- [**PUT /bulkQuotes/**_{ID}_**/error**](#6951-put-bulkquotesiderror)
+- Data Model -- Empty body
+
+#### 6.9.3.2 POST /bulkQuotes
+
+Alternative URI: N/A
+
+Logical API service: **Calculate Bulk Quote**
+
+The HTTP request **POST /bulkQuotes** is used to request the creation of a bulk quote for the provided financial transactions on the server.
+
+Callback and data model information for **POST /bulkQuotes**:
+
+- Callback -- [**PUT /bulkQuotes/**_{ID}_](#6941-put-bulkquotesid)
+- Error Callback -- [**PUT /bulkQuotes/**_{ID}_**/error**](#6951-put-bulkquotesiderror)
+- Data Model -- See [Table 38](#table-38)
+
+###### Table 38
+
+| **Name** | **Cardinality** | **Type** | **Description** |
+| --- | --- | --- | --- |
+| **individualQuoteResults** | 0..1000 | IndividualQuoteResult | Fees for each individual transaction, if any of them are charged per transaction. |
+| **expiration** | 1 | DateTime | Date and time until when the quotation is valid and can be honored when used in the subsequent transaction request. |
+| **extensionList** | 0..1 | ExtensionList | Optional extension, specific to deployment. |
+
+**Table 38 -- POST /bulkQuotes data model**
```

Comment:
Not part of this PR, but we have the wrong data model here (this is the PUT /bulkQuotes/ID model not the POST one, can you please fix it now that you're at it; 

Thank you!

Reply:
Fixed, thanks!